x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.test.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { MockSyncContext } from '../__tests__/mock_sync_context';
+import sinon from 'sinon';
+
+jest.mock('ui/new_platform');
```

Comment:
You can remove this line. It's very copied from something I left behind (sorry) but we don't need it

Reply:
thx!